src/utils/constants.ts

```diff
@@ -4,12 +4,22 @@ import { User } from '@auth0/auth0-react';
 // Local Imports
 import { AppLinkProps } from 'stories/atoms/AppLink/AppLink';
 
+// The original setting of breakpoints leaves the screen size ranged from 0px to 600px defined as Desktop.
+// So instead of using 'min-width', 'max-width' will be more accurate
```

Comment:
Comments like this that aren't going to live in the code forever should be made in your PR, not in the code itself. In-code comments should always be for things that are going to be around after your code merges. If we merged your changes this comment would be difficult to understand without being able to see the diff.  

![Screen Shot 2022-05-21 at 9 26 50 AM](https://user-images.githubusercontent.com/12022922/169653722-40282a2e-dfd8-4807-86e6-2712dc2d6167.png)
👆 Without this `diff` your comment will be hard to understand.